2023-06-27 21:37:37,686        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-27 21:37:37,745        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-27 21:38:15,589        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-27 21:38:15,674        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-27 21:38:27,790        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-27 21:38:27,823        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-27 21:41:42,242        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-27 21:41:42,278        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-27 21:48:41,038        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-27 21:48:41,067        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 21:08:01,396        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 21:08:01,424        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 21:08:22,716        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 49866)
2023-06-28 21:08:22,722        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 49866)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:08:22,722        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687975702.7162752, 'user': {'account_name': 'test_21'}}
2023-06-28 21:08:22,745        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 49866)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:08:23,257        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 49866)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:08:23,257        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687975702.7455401, 'account_name': 'test_21'}
2023-06-28 21:08:23,260        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 49866)>, {'response': 202, 'data_list': ['client_1', 'client_2', 'client_3', 'client_4', 'test_21']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:08:23,765        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 49866)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:08:23,765        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_contacts', 'time': 1687975703.2735238, 'user': 'test_21'}
2023-06-28 21:08:23,771        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 49866)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:08:42,564        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 49866)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:08:42,564        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687975722.2303445, 'account_name': 'test_21'}
2023-06-28 21:08:42,569        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1340, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 49866)>, {'response': 202, 'data_list': ['client_1', 'client_2', 'client_3', 'client_4', 'test_21']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:08:44,615        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1340, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 49866)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:08:44,615        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'add', 'time': 1687975724.1607878, 'user': 'test_21', 'account_name': 'client_2'}
2023-06-28 21:32:41,642        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 21:32:41,670        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 21:32:57,298        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50034)
2023-06-28 21:32:57,300        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1292, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50034)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:32:57,300        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687977177.2984731, 'user': {'account_name': 'client_2'}}
2023-06-28 21:32:57,320        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1292, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50034)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:32:57,825        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1292, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50034)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:32:57,825        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687977177.3203857, 'account_name': 'client_2'}
2023-06-28 21:32:57,828        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1292, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50034)>, {'response': 202, 'data_list': ['client_1', 'client_2', 'client_3', 'client_4', 'test_21']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:32:58,332        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1292, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50034)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:32:58,332        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_contacts', 'time': 1687977177.8372886, 'user': 'client_2'}
2023-06-28 21:32:58,337        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1292, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50034)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:33:47,683        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1292, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50034)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:33:47,684        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687977227.2741125, 'account_name': 'client_2'}
2023-06-28 21:33:47,690        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1292, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50034)>, {'response': 202, 'data_list': ['client_1', 'client_2', 'client_3', 'client_4', 'test_21']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:33:48,700        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1292, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50034)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:33:48,700        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'add', 'time': 1687977228.680785, 'user': 'client_2', 'account_name': 'test_21'}
2023-06-28 21:42:53,295        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 21:42:53,373        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 21:52:49,781        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 21:52:49,808        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 21:53:10,955        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 21:53:10,983        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 21:59:10,765        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 21:59:10,790        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 21:59:27,673        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50451)
2023-06-28 21:59:27,674        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50451)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:59:27,674        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687978767.6737812, 'user': {'account_name': 'client_12'}}
2023-06-28 21:59:27,699        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50451)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:59:28,211        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50451)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:59:28,211        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687978767.6991098, 'account_name': 'client_12'}
2023-06-28 21:59:28,214        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50451)>, {'response': 202, 'data_list': ['client_1', 'client_2', 'client_3', 'client_4', 'test_21', 'client_12']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 21:59:28,722        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50451)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 21:59:28,722        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_contacts', 'time': 1687978768.2245595, 'user': 'client_12'}
2023-06-28 21:59:28,728        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1360, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50451)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:00:16,565        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1360, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50451)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:00:16,565        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'add', 'time': 1687978816.2549412, 'user': 'client_12', 'account_name': 'client_4'}
2023-06-28 22:04:57,425        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 22:04:57,451        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 22:05:16,333        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50590)
2023-06-28 22:05:16,842        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50590)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:05:16,842        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687979116.3583543, 'user': {'account_name': 'client_12'}}
2023-06-28 22:05:16,863        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50590)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:05:17,370        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50590)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:05:17,370        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687979116.8636813, 'account_name': 'client_12'}
2023-06-28 22:05:17,373        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50590)>, {'response': 202, 'data_list': ['client_1', 'client_2', 'client_3', 'client_4', 'test_21', 'client_12']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:05:17,883        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50590)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:05:17,883        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_contacts', 'time': 1687979117.3826504, 'user': 'client_12'}
2023-06-28 22:05:17,889        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1300, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50590)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:05:27,563        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1300, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50590)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:05:27,563        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'add', 'time': 1687979127.5609603, 'user': 'client_12', 'account_name': 'client_4'}
2023-06-28 22:07:52,490        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 22:07:52,520        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 22:08:06,805        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50658)
2023-06-28 22:08:06,807        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1284, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50658)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:08:06,807        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687979286.805017, 'user': {'account_name': 'client_12'}}
2023-06-28 22:08:06,823        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1284, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50658)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:08:07,334        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1284, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50658)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:08:07,335        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687979286.8239632, 'account_name': 'client_12'}
2023-06-28 22:08:07,337        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1284, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50658)>, {'response': 202, 'data_list': ['client_1', 'client_2', 'client_3', 'client_4', 'test_21', 'client_12']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:08:07,850        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1284, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50658)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:08:07,850        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_contacts', 'time': 1687979287.347157, 'user': 'client_12'}
2023-06-28 22:08:07,856        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1284, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50658)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:08:14,995        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1284, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50658)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:08:14,996        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'add', 'time': 1687979294.7829235, 'user': 'client_12', 'account_name': 'client_2'}
2023-06-28 22:08:15,019        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1284, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50658)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:08:49,654        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1284, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50658)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:08:49,655        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'message', 'from': 'client_12', 'to': 'client_2', 'time': 1687979329.5362735, 'mess_text': 'Hi, friends!'}
2023-06-28 22:12:56,888        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 22:12:56,988        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 22:17:04,331        DEBUG        server.py                 Была вызвана функция arg_parser c параметрами ('8079', ''), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-28 22:17:04,358        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-28 22:17:15,141        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50863)
2023-06-28 22:17:15,149        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50864)
2023-06-28 22:17:15,151        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:17:15,151        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687979835.1414125, 'user': {'account_name': 'test1'}}
2023-06-28 22:17:15,171        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:17:15,694        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:17:15,695        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687979835.1716423, 'account_name': 'test1'}
2023-06-28 22:17:15,700        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>, {'response': 202, 'data_list': ['test1']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:17:15,702        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50864)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:17:15,702        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687979835.1494694, 'user': {'account_name': 'test2'}}
2023-06-28 22:17:15,718        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50864)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:17:16,226        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:17:16,226        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_contacts', 'time': 1687979835.7113295, 'user': 'test1'}
2023-06-28 22:17:16,242        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:17:16,243        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50864)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:17:16,243        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687979835.7184284, 'account_name': 'test2'}
2023-06-28 22:17:16,247        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50864)>, {'response': 202, 'data_list': ['test1', 'test2']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:17:16,752        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50864)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:17:16,753        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_contacts', 'time': 1687979836.2537394, 'user': 'test2'}
2023-06-28 22:17:16,758        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50864)>, {'response': 202, 'data_list': []}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:17:40,766        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:17:40,766        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'get_users', 'time': 1687979860.3668902, 'account_name': 'test1'}
2023-06-28 22:17:40,772        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>, {'response': 202, 'data_list': ['test1', 'test2']}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:17:43,836        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:17:43,836        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'add', 'time': 1687979863.5400882, 'user': 'test1', 'account_name': 'test2'}
2023-06-28 22:17:43,868        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:17:53,060        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1320, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50864)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:17:53,060        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'add', 'time': 1687979872.848317, 'user': 'test2', 'account_name': 'test1'}
2023-06-28 22:17:53,071        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=1320, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50864)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции process_client_message
2023-06-28 22:18:13,914        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=1316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50863)>,), {}. Вызов из модуля utils. Вызов из функции self._bootstrap_inner().Вызов из функции run
2023-06-28 22:18:13,914        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'message', 'from': 'test1', 'to': 'test2', 'time': 1687979893.254608, 'mess_text': 'Hi, my friends!'}
